src/MicrosoftTeams.schema.json

```diff
@@ -92,17 +98,20 @@
         },
         "accentColor": {
             "type": "string",
-            "description": "A color to use in conjunction with the icon. The value must be a valid HTML color code starting with '#', for example `#4464ee`."
+            "description": "A color to use in conjunction with the icon. The value must be a valid HTML color code starting with '#', for example `#4464ee`.",
+            "pattern": "^#[0-9a-fA-F]{6}$"
```

Comment:
Nice, we should also update this in the settings store which today only enforces a max length of 7 for the accentColor.